File: scala-gatling/src/gatling/scala/org/openapitools/client/model/JsonErrorBaseAllOf.scala

package org.openapitools.client.model


case class JsonErrorBaseAllOf (
    _result: String,
    _msg: String
)
object JsonErrorBaseAllOf {
    def toStringBody(var_result: Object, var_msg: Object) =
        s"""
        | {
        | "result":$var_result,"msg":$var_msg
        | }
        """.stripMargin
}

File: kotlin-spring/src/main/kotlin/org/openapitools/model/JsonErrorBaseAllOf.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonValue
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param result 
 * @param msg 
 */
data class JsonErrorBaseAllOf(

    @field:JsonProperty("result", required = true) val result: JsonErrorBaseAllOf.Result,

    @field:JsonProperty("msg", required = true) val msg: kotlin.String
) {

    /**
    * 
    * Values: error
    */
    enum class Result(val value: kotlin.String) {
    
        @JsonProperty("error") error("error");
    
    }

}

